crates/flipperzero/src/dialogs/mod.rs

```diff
@@ -25,6 +25,7 @@ pub struct DialogMessage<'a> {
 }
 
 /// A dialog file browser options.
+#[repr(transparent)]
 pub struct DialogFileBrowserOptions<'a> {
```

Comment:
I note that this new struct was not documented as an addition in the changelog. As we haven't cut a release with it yet, we don't need to note the changes in this PR; just add the entire struct as an addition.

Reply:
Good point, added it